### PR TITLE
Pipe changs 

### PR DESCRIPTION
**Pull request by guyyaffear**

### CODE REVIEW

File: README.md
Comment (lavie707) [inline README.md:1]: Mayme we need to write our own README? because there is a lot of links to httptoolkit's github

File: README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
-HTTP Toolkit UI [![Build Status](https://github.com/httptoolkit/httptoolkit-ui/workflows/CI/badge.svg)](https://github.com/httptoolkit/httptoolkit-ui/actions)
+Pipe UI [![Build Status](https://github.com/httptoolkit/httptoolkit-ui/workflows/CI/badge.svg)](https://github.com/httptoolkit/httptoolkit-ui/actions)
```
Comment (lavie707) [inline README.md]: For example, here the URL is still `https://github.com/httptoolkit/httptoolkit-ui......`

File: src/components/account/plan-picker.tsx [hunk 2]
```diff
@@ -310,7 +310,7 @@ export class PlanPicker extends React.Component<PlanPickerProps> {
                     <p>
                         By subscribing to a paid plan, you accept <Nowrap>
                             <a href="https://httptoolkit.com/terms-of-service">
-                                the HTTP Toolkit Terms of Service
+                                the Pipe Terms of Service
                             </a>
```
Comment (lavie707) [inline src/components/account/plan-picker.tsx]: Why we need Terms of service lets delete it.. it also redirect to httptoolkit website

File: src/components/account/pro-placeholders.tsx [hunk 3]
```diff
@@ -66,7 +66,7 @@ export const CardSalesPitch = inject('accountStore')(observer((p: {
 
     <GetProButton onClick={() => p.accountStore!.getPro(p.source)}>
         <Icon icon={['far', 'star']} size='2x' />
-        Get HTTP Toolkit Pro
+        Get Pipe Pro
```
Comment (lavie707) [inline src/components/account/pro-placeholders.tsx:69]: Do we still have `Pro` mode and `basic` mode?

File: src/components/app.tsx [hunk 4]
```diff
@@ -120,35 +120,6 @@ class App extends React.Component<{ accountStore: AccountStore }> {
...
-
-            {
-                name: 'Give feedback',
-                title: "Suggest features or report issues",
-                icon: ['far', 'comment'],
-                position: 'bottom',
-                highlight: true,
-                type: 'web',
-                url: 'https://github.com/httptoolkit/httptoolkit/issues/new/choose'
-            }
```
Comment (lavie707) [inline src/components/app.tsx:151]: If we dont have pro mode its ok, if not we need to remain this code

File: src/images/logo-icon.svg
Comment (lavie707) [inline src/images/logo-icon.svg:1]: America!

File: src/index.html [hunk 5]
```diff
@@ -123,7 +129,7 @@
                 if (error.isTimeout) {
                     errorElement.innerText = 'This is taking longer than normal...';
                     errorDetailsElement.innerHTML = 'If this persists or happens often ' +
-                        'please <a href="https://github.com/httptoolkit/httptoolkit/issues/new">file an issue</a> ' + 'and help make HTTP Toolkit faster.';
+                        'please <a href="https://github.com/httptoolkit/httptoolkit/issues/new">file an issue</a> ' + 'and help make Pipe faster.';
```
Comment (lavie707) [inline src/index.html]: delete it

File: README.md
Comment (lavie707) [reply README.md:1]: Maybe*

Comment (guyyaffear) [reply README.md:1]: i removed it

File: src/components/account/pro-placeholders.tsx (hunk 3 above)
Comment (guyyaffear) [reply src/components/account/pro-placeholders.tsx:69]: remove it

File: src/components/account/plan-picker.tsx (hunk 2 above)
Comment (guyyaffear) [reply src/components/account/plan-picker.tsx]: set it in command out

File: src/components/app.tsx (hunk 4 above)
Comment (guyyaffear) [reply src/components/app.tsx:151]: i set it in command out

File: src/index.html (hunk 5 above)
Comment (guyyaffear) [reply src/index.html]: done

**Pull request merged**